src/core/appdir.cpp

```diff
@@ -673,11 +675,11 @@ namespace linuxdeploy {
             }
 
             bool AppDir::deployLibrary(const bf::path& path, const bf::path& destination) {
-                return d->deployLibrary(path, false, true, destination);
+                return d->deployLibrary(path, d->excludeLibraryPatterns, false, true, destination);
```

Comment:
No need to change the method's signature. `PrivateData` has access to its own members. You do not need to pass its own data to its methods.

Reply:
derp... for some reason I thought PrivateData::deployLibrary() was a static method.